notebooks/233-encodec-audio-compression/233-encodec-audio-compression.ipynb

```diff
@@ -29,7 +48,32 @@
     }
    ],
    "source": [
-    "!pip install encodec"
+    "!pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82279f60-a018-4ea8-9b0b-77f0f40903cf",
+   "metadata": {},
+   "source": [
+    "## Instantiate audio compression pipeline\n",
+    "\n",
+    "[Codecs](https://en.wikipedia.org/wiki/Codec), which act as encoders and decoders for streams of data, help power most of the audio compression people currently use online. Some examples of commonly used codecs include MP3, Opus, and EVS. Classic codecs like these decompose the signal between different frequencies and encode as efficiently as possible. Most classic codecs leverage human hearing knowledge (psychoacoustics) but have a finite or given set of handcrafted ways to efficiently code and decode the file. EnCodec, a neural network that is trained end to end to reconstruct the input signal, was introduced as attempt to overcome this limitation. It consits of 3 parts:\n",
+    "\n",
+    "* The **encoder**, which takes the uncompressed data and transforms it into a higher dimensional and lower frame rate representation.\n",
+    "\n",
+    "* The **quantizer**, which compresses this representation to the target size. This compressed representation is what is stored on disk or will be send through the network.\n",
```

Comment:
* The **quantizer**, which compresses this representation to the target size. This compressed representation is what is stored on disk or will be sent through the network.